src/apb/engine.py

```diff
@@ -29,9 +32,44 @@ def load_dockerfile(df_path):
         return dockerfile.readlines()
 
 
-def load_example_specfile():
-    with open(EX_SPEC_FILE_PATH, 'r') as spec_file:
-        return spec_file.readlines()
+def convert_params_to_dict(params):
+    param_dict = {}
+    param_list = []
+
+    for param in params:
```

Comment:
How is ```params``` being passed around? It is in the format of ```[item=value, item2=value2, ect...]```?  So that would be a list of strings. Maybe there's a better way to read in the params so we don't have to convert it.